### DISCUSSION TITLE
How to cluster by a specific field (eg. zip code) of input data?

### DISCUSSION THREAD

**Discussion by xhy1286** [visgl/flowmap.gl#31] [Q&A] [answered]
I have a flow data which contains zip codes of origins and destinations, I want to cluster by zip code instead of by distance... but I don't know how to realize it based on flowmap.gl

**Comment by ilyabo** [comment, marked as answer]
Sorry for the delay. It should be possible by providing custom `clusterLevels` as is done [here for the H3-based clustering](https://github.com/visgl/flowmap.gl/blob/upgrade-deps/examples/common/src/fetchData.js#L37)
